python/server/server_auto.sip

```diff
@@ -1,38 +1,21 @@
 // Include auto-generated SIP files
-%Include auto_generated/qgsservicemodule.sip
-%Include auto_generated/qgsmapserviceexception.sip
-%Include auto_generated/qgscapabilitiescache.sip
-%Include auto_generated/qgsconfigcache.sip
-%Include auto_generated/qgsserverlogger.sip
-%Include auto_generated/qgsserversettings.sip
-%Include auto_generated/qgsserverparameters.sip
 %Include auto_generated/qgsbufferserverrequest.sip
 %Include auto_generated/qgsbufferserverresponse.sip
+%Include auto_generated/qgscapabilitiescache.sip
```

Comment:
What's happened here? I don't think the changes to this file area required.

Reply:
Travis asked me to regenerate the sip... I didn't pay attention to this change, I'll change it manually.